.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        otp_version: ['26.1', '25.3', '24.3']
+        otp_version: ['26.2', '25.3']
```

Comment:
@whatyouhide do you want to test on 27 too? Or it's too big of a jump for this PR.

Reply:
I wanted to do that in a separate PR to avoid mixing things up.